Modules/Core/Common/test/itkPeriodicBoundaryConditionTest.cxx

```diff
@@ -308,9 +308,9 @@ itkPeriodicBoundaryConditionTest(int, char *[])
     for (auto & j : back)
     {
       const float d = testIter.GetPixel(j);
-      sum += d * d;
     }
     ++testIter;
+    (void)sum;
```

Comment:
I guess logging to console would work, preferably just once here instead of multiple times in the loop.

Reply:
OK, I now use this dummy sum and log it.